ui/Config.ui

```diff
@@ -48,7 +48,7 @@
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>2</number>
```

Comment:
This makes the color config tab the active tab the first time you open the config dialog (you may have changed this accidentally by browsing the config dialog tabs in QtCreator). Just set it back to zero and that would be it.